### DISCUSSION TITLE
[Feature] <feature-name-goes-here>

### DISCUSSION THREAD

**Discussion by soccer1902** [immich-app/immich#9161] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Customize the size of the cache on server to speed up frequent content lookup

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
There is no (size-limited) cache on the server.

**Reply by soccer1902** [reply]
I'm thinking about a tiered storage approach, where more frequent assets are stored on faster drives If possible

**Reply by bo0tzz** [reply]
The full size assets are not usually accessed while browsing the app. You can store just the generated files on SSD to speed things up a bit (https://immich.app/docs/guides/custom-locations). Building multi-tier cache management into Immich seems overkill, you can manage that on a filesystem level if you really need it.
